Edi/Edi.Apps/Behaviors/ActivatedCommand.cs

```diff
@@ -17,7 +17,7 @@ public static class ActivatedCommand
 				"Command",
 				typeof(ICommand),
 				typeof(ActivatedCommand),
-				new PropertyMetadata(null, ActivatedCommand.OnCommandChange));
```

Comment:
I don't really see an improvement here (just a different preferedcoding style but I don't mind using this style instead)